meteor/client/ui/SegmentTimeline/SmallParts/SegmentTimelineSmallPartFlag.tsx

```diff
@@ -114,6 +116,7 @@ export const SegmentTimelineSmallPartFlag = ({
 				isLastInSegment={isLastInSegment}
 				totalSegmentDuration={partDurations}
 				parentTimeScale={timeScale}
+				showDurationSourceLayers={showDurationSourceLayers}
```

Comment:
I think we should make a point to refactor `showDurationSourceLayers` _et consortes_ into some sort of a React context. I hate how this needs to be passed around like that. Not something to be addressed right now, but soon-ish.

Reply:
Let's use some central state management. MobX perhaps? 😃

Reply:
To be continued then.. :)